### PR TITLE
Setting up photos

### PR DESCRIPTION
**Pull request by lcchou2**

### CODE REVIEW

File: Photos/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "react-dev": "webpack -d --watch",
+    "start-dev": "nodemon index.js",
+    "build": "webpack -d",
+    "start": "node index.js"
+  },
+  "author": "Beth Johnson",
```
Comment (MichaelHamm) [inline Photos/package.json]: You are not beth

File: Photos/client/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+<head>
+<script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+<script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+
+</head>
+
+
+<body>
+
+<div id = 'app'>
```
Comment (iyshao) [inline Photos/client/index.html]: please indent!

File: Photos/package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "react-dev": "webpack -d --watch",
+    "start-dev": "nodemon server/index.js",
+    "build": "webpack -d",
+    "start": "node server/index.js"
+  },
+  "author": "Mike Hamm sucks",
```
Comment (MichaelHamm) [inline Photos/package.json]: :0

**Review by iyshao — APPROVED** [review]

**Review by Mehul2001 — APPROVED** [review]
good

**Review by MichaelHamm — APPROVED** [review]

**Pull request merged**